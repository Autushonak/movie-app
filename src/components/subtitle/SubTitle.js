import styled from 'styled-components';
import PropTypes from 'prop-types';
import { colors } from '../../constants/colors';

const SubTitle = styled.h4`
  text-transform: uppercase;
  color: ${colors.WHITE};
  margin-right: 10px;
`;

SubTitle.propTypes = {
  children: PropTypes.string,
};

export default SubTitle;
