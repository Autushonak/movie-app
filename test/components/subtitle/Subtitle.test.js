import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import SubTitle from '../../../src/components/subtitle/SubTitle';

describe('SubTitle ', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<SubTitle>Any subtitle</SubTitle>);
    expect(component).toMatchSnapshot();
  });
});
