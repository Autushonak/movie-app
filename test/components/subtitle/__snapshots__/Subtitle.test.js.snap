// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitle  should be rendered correctly 1`] = `
.c0 {
  text-transform: uppercase;
  color: #FFFFFF;
  margin-right: 10px;
}

<h4
  className="c0"
>
  Any subtitle
</h4>
`;
